select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -465,4 +465,12 @@ public void onComponentTagBody(final MarkupStream markupStream, final ComponentT
         }
 		replaceComponentTagBody(markupStream, openTag, buffer);
 	}
+	
+	/**
+	 * Empty input is acceptable
+	 */
+	@Override
+	public boolean isInputNullable() {
+		return true;
+	}
```

Comment:
For some reason https://github.com/apache/wicket/blob/be18152c3d548af9e747d32385aa1f03f113cdbd/wicket-core/src/main/java/org/apache/wicket/markup/html/form/AbstractSingleSelectChoice.java#L45 has value "false" by default
I believe it will be good idea to have "false" here
@martin-g WDYT ?